lib/engine/step/g_18_zoo/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../buy_sell_par_shares'
+
+module Engine
+  module Step
+    module G18ZOO
+      class BuySellParShares < BuySellParShares
+        def actions(entity)
```

Comment:
why did you need to implement this method?

Reply:
the original contains
`actions << 'buy_company' unless purchasable_companies(entity).empty?`
but the `purchasable_companies` is used for both showing the buyable companies on the top and on the bottom
<img width="1680" alt="Screenshot 2021-01-31 at 21 49 06" src="https://user-images.githubusercontent.com/1408567/106570325-02876880-6536-11eb-84b3-94ae6f16cbfe.png">
I don't want to add `can_buy_companies_from_other_players` but without that I'll never be able to show the companies.

Maybe we could change the behaviour to be able to buy from bank without `can_buy_companies_from_other_players` ?